generator/AWSPSGeneratorLib/HelpMaterials/Examples/DS/Start-DSSchemaExtension.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$ldif = gc D:\Users\jvadaria\Downloads\ExtendedSchema.ldf -raw
+Start-DSSchemaExtension -DirectoryId d-123456ijkl -CreateSnapshotBeforeSchemaExtension $true -Description ManagedADSchemaExtension -LdifContent $ldif
+e-9067306643</code>
```

Comment:
I wonder what is c-9067306643 here? If it is output, then let's add a new line between the command and output.

Also, can you update the gc with full command?

```
$ldif= Get-Content -Path schema.ldif -Raw
Start-DSSchemaExtension -DirectoryId d-123456ijkl -CreateSnapshotBeforeSchemaExtension $true -Description ManagedADSchemaExtension -LdifContent $ldif
```

Reply:
Updated.